### DISCUSSION TITLE
Cypress tests on schedule

### DISCUSSION THREAD

**Discussion by RuthlessB** [cypress-io/cypress#21559] [Questions and Help]
Greetings, fellow testers!

I am quite new to testing and Cypress. But let's say I have a few tests, which I would like to run once every 30 minutes or so. The Cypress test laucher only allows one run, after which I have to start another one manually, Is there any solution for this type of task, which is not paid (like browserstack and what not)? Thank you in advance for helping me out.

**Comment by muratkeremozcan** [comment]
You need a cron job. Every CI provider supports it. https://crontab.guru/#0_1-23_*_*_6-7

Setup Cypress to be able to run in the CI, then implement the cron job.

**Reply by RuthlessB** [reply]
Hello, Murak. Huge thanks for this useful info. Will try it out for sure.

**Reply by hatemeby** [reply]
Hello @RuthlessB. I was just wondering if that worked for you. If yes, can you give a bit details about some specific moment or some info which will be useful. Thank you!
